pyesbulk/__init__.py

```diff
@@ -179,6 +278,11 @@ def put_template(es, name=None, mapping_name=None, body=None):
                 # All other error codes are some kind of error we don't attempt
                 # to retry.
                 raise
+        except Exception as e:
+            logger.error(
+                "Unexpected exception checking %s: %r", name, e
+            )
+            raise
```

Comment:
We might want to consider in the future adding an `Exception` class as part of the interface for this condition.